CMakeLists.txt

```diff
@@ -2932,19 +2934,19 @@ if(APPLE AND MACOS_BUNDLE)
             endif()
         endmacro()
 
-        install_qt5_plugin(Qt5::QCocoaIntegrationPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QMacStylePlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QOffscreenIntegrationPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QMinimalIntegrationPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QSvgPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QJpegPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QGifPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QICOPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
-        install_qt5_plugin(Qt5::QSQLiteDriverPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
+        install_qt5_plugin(QCocoaIntegrationPlugin BUNDLE_LIBS "${MIXXX_INSTALL_PREFIX}")
```

Comment:
Should this macro be renamed to `install_qt_plugin` without any version number?

Reply:
Should we just remove it now that we're not using it?

Reply:
The rest of the build system is written with the assumption that a dynamic build should continue to work. 
In this sense I would keep it, even though, we can't verify it and it will suffer rotting.
But it will definitely be helpfull in case we want to switch to a dynamic build later.

Reply:
I think it would be better to remove it. Yes, we need to support dynamic linking, but we don't need to do so for macOS.

Reply:
@uklotzde what do you think about removing this no longer used code?

Reply:
Ok. If it is not used then we should probably remove it. Less and simpler is better.